.nvmrc

```diff
@@ -0,0 +1 @@
+8
```

Comment:
Please remove this file. Your PR says "Update Readme". Why do we need to have this file?

If this is needed, have a separate PR for it. I don't see anything wrong with having a one-line or one-character change PR. It makes the commit history comprehensive

Reply:
Have removed it, but based on @adesege comment, it would be required to inform the version of node should be run